2016/08/16 07:27:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_89_2_1411448132.jpg".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_89_2_1411448132.jpg
HTTP_REFERER=http://localhost:81/mypropertylk/list/property/type/buy
---
2016/08/16 07:27:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "upload/propertyimages/image_89_4_1411448133.jpg".' in C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php:280
Stack trace:
#0 C:\xampp\htdocs\mypropertylk\protected\framework\web\CWebApplication.php(135): CWebApplication->runController('upload/property...')
#1 C:\xampp\htdocs\mypropertylk\protected\framework\base\CApplication.php(162): CWebApplication->processRequest()
#2 C:\xampp\htdocs\mypropertylk\index.php(15): CApplication->run()
#3 {main}
REQUEST_URI=/mypropertylk/upload/propertyimages/image_89_4_1411448133.jpg
HTTP_REFERER=http://localhost:81/mypropertylk/list/property/type/buy
---
